Unix/installbuilder/GNUmakefile

```diff
@@ -141,10 +141,6 @@ ifneq ($(PF),Darwin)
 		--DATAFILE_PATH=$(TOP)/installbuilder/datafiles \
 		$(DATAFILES)
 
-    ifeq ($(COMPRESS_KIT),1)
-	cd $(OUTPUTDIR)/release; compress -f `cat package_filename`
```

Comment:
this is not duplicate, package_filename content in Line 145 is `omi-1.4.2-105.ssl_110.ulinux.x64.withsymbols.deb`, and package_filename content in Line 172 is `omi-1.4.2-105.ssl_110.ulinux.x64.deb`